simulation/halsim_ws_client/src/main/native/cpp/HALSimWS.cpp

```diff
@@ -64,9 +64,11 @@ bool HALSimWS::Initialize() {
   if (uri != NULL) {
     m_uri = uri;
   } else {
-    m_uri = "/wpilibws";
+    m_uri = "/wpilibws/halsimws";
```

Comment:
This is more a note for @jpokornyiii and I: We should update documentation to specify that clients will connect to a server URI of the form `/<base URI>/<identifier>`. We'll need to update our reference implementations accordingly too